### ISSUE TITLE
Bug: Category is not verified when creating problems

### ISSUE THREAD

**Issue by breeswish** [vijos/vj4#166]
According to designs, problem categories should be limited to our built-in categories for better problem organization across domains.

https://github.com/vijos/vj4/issues/163

**Comment by twd2** [comment]
Why is this a problem?

**Comment by breeswish** [comment]
@twd2  Repeat: According to designs, problem categories should be limited to our built-in categories for better problem organization across domains.

**Comment by iceb0y** [comment]
@breeswish I don't agree - user can use vj4/vijos for something very different than OI

**Comment by breeswish** [comment]
We can make this into a convenience feature - user is allowed to customize category with some effort so that for normal users they are encouraged to use our built-in categories.
